### ISSUE TITLE
SyntaxError: The requested module '@llamaindex/cloud/api' does not provide an export named 'client'

### ISSUE THREAD

**Issue by JeremyKirkham** [run-llama/LlamaIndexTS#1176]
When compiling and running the RAG example, I get the following error:

```
> node dist/index.js

file:///Users/jeremykirkham/projects/ai/node_modules/llamaindex/dist/cloud/utils.js:1
import { client, ProjectsService } from "@llamaindex/cloud/api";
         ^^^^^^
SyntaxError: The requested module '@llamaindex/cloud/api' does not provide an export named 'client'
```

`npm ls:`

```
jeremykirkham@Jeremys-MacBook-Air ai % npm ls @llamaindex/cloud
ai@1.0.0 /Users/jeremykirkham/projects/ai
└─┬ llamaindex@0.5.23
  └── @llamaindex/cloud@0.2.2
```

**Comment by AndreMaz** [comment]
I can confirm the error.
@himself65 this started to happen after yesterday's fix of https://github.com/run-llama/LlamaIndexTS/issues/1172

**Comment by JeremyKirkham** [comment]
This looks fixed by `0.5.24`
